internal/auth/oidc/testing.go

```diff
@@ -176,6 +176,24 @@ func TestAccount(t *testing.T, conn *gorm.DB, am *AuthMethod, subject string, op
 	return a
 }
 
+// TestManagedGroup creates a test oidc managed group.
+func TestManagedGroup(t *testing.T, conn *gorm.DB, am *AuthMethod, filter string, opt ...Option) *ManagedGroup {
```

Comment:
mind adding some tests that would cover different scenarios where ManagedGroup.validate(...) would fail?

Reply:
Done.